docs/queries/filters.md

```diff
@@ -43,22 +43,18 @@ When the day changes, relative dates like `due today` are re-evaluated so that t
 
 ## Matching - Workaround for Boolean OR
 
-All filters of a query have to match in order for a task to be listed.
-This means you cannot show tasks that have "GitHub in the path and have no due date or are due after 2021-04-04".
-Instead you would have two queries, one for each condition:
-
-    ### Not due
+Each line of a query has to match in order for a task to be listed.
+In other words, lines are considered to have an 'AND' operator between them.
+Within each line, you can use the boolean operators AND, OR, AND NOT, OR NOT and XOR, as long as individual filters are wrapped in parenthesis or quotes:
```

Comment:
The docs say that 'XOR' is supported, but when I write a test that uses it:

```typescript
        const filter = new BooleanField().createFilterOrErrorMessage(
            '(description includes d1) XOR (description includes d2)',
        );
```

I get this error:

`Error: Unsuppoted operatorXOR`

- The docs and code are inconsistent 
- Typo: Unsuppoted
- Really nice that the problem XOR is in the error message - maybe add a space beforehand?

Reply:
Oops, I didn't actually implement it in `filterTaskWithParsedQuery`.
Will add the few lines and the test for it.

Reply:
(done)